src/index.html

```diff
@@ -5,8 +5,430 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header header--margins">
+      <a class="header__img-link" href="#home">
+        <img
+          class="header__img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list" >
+          <li class="header__list">
+            <a class="header__link is-active" href="/">Apple</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="header__list">
+            <a
+              class="header__link"
+              data-qa="nav-hover"
+              href="#lap&Comp"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="photo">Photo</a>
+          </li>
+
+          <li class="header__list">
+            <a class="header__link" href="video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="product">
+        <div class="product-card" data-qa="card">
```

Comment:
it doesn't match bem naming convention

`product-card, product-card-image... and so on`

fix all such cases according to bem